### DISCUSSION TITLE
Is there a way to trigger argo workflow and rollouts on remote k8s cluster with events sensor?

### DISCUSSION THREAD

**Discussion by riuvshyn** [argoproj/argo-events#2221] [Q&A]
My use case I have centralised ArgoCD setup that managed many remote clusters so I know that both workflows and rollouts controllers are cluster local only so I can install them on remote clusters
but I want to orchestrate CD pipeline which will do something like: listen to GitHub event -> sensor triggers workflow on a remote cluster1 -> sensor triggers rollout on a remote cluster1 -> sensor triggers workflow on a remote cluster2 -> sensor triggers rollout on a remote cluster2
so the events is installed on a cluster whey ArgoCD.
Or should I deploy dedicated instance of argo events on every remote cluster too? but in that case I can’t orchestrate multi cluster rollout which is my actual goal

**Comment by haseebshaikh48** [comment]
I do have a similar sort of use case. I'm having multiple cluster which are being managed by one centralized cluster where i've Argo Apps installed (ArgoCD Argo events).  I want to trigger workflow in those remote cluster by the sensor which is running inside the centralized cluster.

**Comment by whynowy** [comment]
This is not supported now, maybe submit an enhancement request?
